File: Day-07-of-30-Days/Solution.js
// Day 07 of 30 days 👾

function sumOfIntegers(num) {
    let sum = 0;
    for (let i = 1; i <= num; i++) {
        sum += i;
    }
    return sum;
}

// example 1
console.log(sumOfIntegers(5)); // ans: 15

// example 2
console.log(sumOfIntegers(10)); // ans: 55

File: Day-07-of-30-Days/Solution.py
# Day 07 of 30 Days 👾
def sum_of_integers(num):
    sum = 0
    for i in range(1, num+1):
        sum += i
    return sum

# Example 1
print(sum_of_integers(5)) # ans: 15

# Example 2
print(sum_of_integers(10)) # ans: 55